docs/source/api/apollo-server.mdx

```diff
@@ -457,203 +414,93 @@ The default value is `true`. Set this to `false` to use mocked resolvers only fo
-| Micro | <code>{ req: [`MicroRequest`](https://github.com/apollographql/apollo-server/blob/c356bcf3f2864b8d2fcca0add455071e0606ef46/packages/apollo-server-micro/src/types.ts#L3-L5), res: [`ServerResponse`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/50adc95acf873e714256074311353232fcc1b5ed/types/node/v10/http.d.ts#L145-L158) }</code> |
+The async `start` method instructs Apollo Server to prepare to handle incoming operations.
 
-## `listen`
+> Call `start` **only** if you are using a framework integration for a non-serverless environment (e.g., [`expressMiddleware`](./express-middleware)).
+> * If you're using a serverless framework integration (such as Lambda), you **shouldn't** call this method before passing your server into the integration function. Your serverless integration function will take care of starting your server for you.
+>
+> * If you're using `startStandaloneServer` you do **not** need to start your server before [passing it directly to `startStandaloneServer`](./standalone).
 
-> This method is provided only by the `apollo-server` package. If you're integrating with Node.js middleware via a different package (such as `apollo-server-express`), instead see both [`start`](#start) and the [framework-specific middleware function](#framework-specific-middleware-function).
+Always call `await server.start()` *before* passing your server into your integration function and starting to accept requests:
 
-Instructs Apollo Server to begin listening for incoming requests:
+<MultiCodeBlock>
 
-```js
-await server.listen({
-  port: 4001,
+```ts
+const server = new ApolloServer<MyContext>({
+  typeDefs,
+  resolvers,
 });
-```
-
-Takes an `options` object as a parameter, which is passed to the `listen` method of `http.Server`. Supported options are listed in the [documentation for `net.Server.listen`](https://nodejs.org/api/net.html#net_server_listen_options_callback).
-
-Returns a `Promise` that resolves to an object containing the following properties:
-
-<table class="field-table">
-  <thead>
-    <tr>
-      <th>Name /<br/>Type</th>
-      <th>Description</th>
-    </tr>
-  </thead>
-
-<tbody>
-<tr>
-<td>
-
-##### `url`
-
-`string`
-</td>
-<td>
-
-The URL that the server is listening on.
-</td>
-</tr>
+await server.start();
 
+app.use('/graphql',
+  cors<cors.CorsRequest>(),
+  json(),
+  expressMiddleware(server),
+);
+```
 
-<tr>
-<td>
+</MultiCodeBlock>
 
-##### `server`
+This allows you to react to Apollo Server startup failures by crashing your process instead of starting to serve traffic.
 
-[`http.Server`](https://nodejs.org/api/http.html#http_class_http_server)
-</td>
-<td>
-
-The server instance that's listening at `url`.
-</td>
-</tr>
+If you are testing your server using [`executeOperation`](../testing/testing/) (i.e., you're not actually starting an HTTP server), you still need to call `start()` because `executeOperation` will *not* do that for you.
```

Comment:
That's not true; for convenience in testing, `executeOperation` does call `start` for you.

Reply:
So, in the Integration Testing article, we say:

> If you are using the `startStandaloneServer` function, you don't need to start your server before calling `executeOperation`. The server instance will start automatically and throw any startup errors. If you are using an Apollo Server integration (e.g., `expressMiddleware`), you do need to start your server before calling the `executeOperation` method.

...which makes no sense, because you are right; we don't need to call `start` on the server, so my bad! I'll fix this here and in the integration testing article.